app/controllers/concerns/act_as_api_request.rb

```diff
@@ -3,12 +3,14 @@ module ActAsApiRequest
 
   included do
     before_action :skip_session_storage
-    before_action :check_json_request
+    before_action :check_request_type
   end
 
-  def check_json_request
+  def check_request_type
     return if request_body.empty?
-    return if request.content_type&.match?(/json/)
+
+    content_type = request.content_type
+    return if content_type&.match?(/json/) || content_type&.match?(/form-data/)
```

Comment:
Could we do an array of formats and check against that?

Reply:
I could but I think the solution is not very readable.

example:
`allowed_types = %w[json form-data]
    content_type = request.content_type

    return if allowed_types.any? { |word| content_type.include?(word) }`

or 

`allowed_types = %w[json form-data]
    content_type = request.content_type

    return if content_type.match(Regexp.union(allowed_types))`

WDYT?

Reply:
And what about making it one regex? What I don't like is that you have to add another`|| content_type&.match?(/another-format/)` if you add another format in the future, it would start to get messy.

Reply:
IMHO, if you're gonna handle only 2 content types I would leave it as it is

Reply:
the point of this boilerplate is that it's a json API, I don't think there would be much more request types and if so, it wouldn't be something common. If you still think it's better with a regex I could use the second solution I described in my comment above.

Reply:
@lucaleivaloop thoughts?

Reply:
As it is will do.